server/utils/remote_constants.py

```diff
@@ -0,0 +1,2 @@
+REMOTE_NODE1_HOST = "https://deployservertest.herokuapp.com/"
+REMOTE_NODE2_HOST = "https://website404.herokuapp.com/"
```

Comment:
We have the remote node DB table for this

Reply:
I did this because there was a minor difference between the 2.